cli/restart_test.go

```diff
@@ -127,7 +127,7 @@ func TestRestart(t *testing.T) {
 		coderdtest.AwaitWorkspaceBuildJobCompleted(t, client, workspace.LatestBuild.ID)
 
 		inv, root := clitest.New(t, "restart", workspace.Name,
-			"--build-option", fmt.Sprintf("%s=%s", ephemeralParameterName, ephemeralParameterValue))
```

Comment:
We might want one test to validate that `--build-option` also still works.

Reply:
Just added back a test for both `--build-option` and `--build-options`. 🙌